src/main/java/com/wiley/assertions/TeasyError.java

```diff
@@ -16,7 +16,14 @@ public TeasyError(Throwable throwable, MethodType methodType) {
     }
 
     public String getErrorMessage() {
-        String message = throwable != null && throwable.getMessage() != null ? throwable.getMessage() : throwable != null ? throwable.getCause().getMessage() : null;
+        String message;
+        if (throwable != null && throwable.getMessage() != null) {
+            message = throwable.getMessage();
+        } else if (throwable != null && throwable.getCause() != null) {
+            message = throwable.getCause().getMessage();
+        } else {
```

Comment:
seems that this else could be removed. (null is assigned by default if im not wrong)